cockroach/Makefile

```
@@ -12,9 +12,12 @@ build-22.1.0-beta.2:
 build-22.1.0-beta.5:
 	docker build -t prismagraphql/cockroachdb-custom:22.1.0-beta.5 ./22.1.0-beta.5
 
+build-22.1.0-rc.1:
+	docker build -t prismagraphql/cockroachdb-custom:22.1.0-rc.1 ./22.1.0-rc.1
 
 push:
 	docker push prismagraphql/cockroachdb-custom:21.2.0-patched
 	docker push prismagraphql/cockroachdb-custom:22.1.0-alpha.1
 	docker push prismagraphql/cockroachdb-custom:22.1.0-beta.2
 	docker push prismagraphql/cockroachdb-custom:22.1.0-beta.5
+	docker push prismagraphql/cockroachdb-custom:22.1.0-rc.1
```

How about we already use `22.1.0` and just keep updating this when new RCs and later release version come in?

```suggestion
build-22.1.0-rc.1:
	docker build -t prismagraphql/cockroachdb-custom:22.1.0 ./22.1.0

push:
	docker push prismagraphql/cockroachdb-custom:21.2.0-patched
	docker push prismagraphql/cockroachdb-custom:22.1.0-alpha.1
	docker push prismagraphql/cockroachdb-custom:22.1.0-beta.2
	docker push prismagraphql/cockroachdb-custom:22.1.0-beta.5
	docker push prismagraphql/cockroachdb-custom:22.1.0
```

(Will also require renaming the folder manually)

---

sounds good to me, making that change